index.html

```diff
@@ -152,45 +152,747 @@
+          important activities move to the Internet, <a>entities</a> need to be able to
+          transmit instantly <em>verifiable</em> claims (e.g., about their location,
+          accomplishments, value, what-have-you). From educational records to payment
+          account access, the next generation of web applications will authorize
+          <a>entities</a> to perform actions based on rich sets of credentials issued by
+          trusted parties. The Holders of these claims, in the form of Verfiable Credentials,
+          need to be able to take the claims they control and make use of them where, when,
+          and how they wish, often in ways the original issuer of the VC may not have 
+          originally indended. Having a common, standard way for these Holders of claims
+          to use them as they wish is vital to a person having control of their own data. 
+      </p>
       <p>
-        TODO: fill in
+          Standardization of the transport layer for digital <a>claims</a> makes it possible for many
+          stakeholders to interact with their desired counterparties, without being locked into proprietary platforms.
+      </p>
+    </section>
+    <section>
+      <h3>Use case model</h3>
+      <p>
+          This document presents an aggregate use case model, comprised of Needs, Roles,
+          Tasks, and Sequences. Taken together, these models define the use cases that
+          the VC-API Working Group has addressed.
+      </p>
+      <p>
+          User needs define the problem space addressed by <a>Verifiable Credentials</a>.
+          User Roles specify the roles different <a>entities</a> play when interacting
+          with <a>Verifiable Credentials</a>. Tasks define the functions users can
+          accomplish, and sequences demonstrate how tasks might be realized, by
+          interactions between <a>entities</a> over time.
+      </p>
+      <p>
+          As with all models, this use case model is neither exhaustive nor complete. The
+          listed uses cannot capture all possible use cases. Similarly, the models do not
+          completely characterize the use cases represented. However, the combined model
+          is intended to provide specific, coherent guidance for the work ahead.
+      </p>
+      <p>
+        The model used here draws heavily from the Verifiable Credentials Use Cases model.
+        This is due to the fact that the VC-API is meant to support the transport of 
+        VCs in the use cases considered for the Verfiable Credential Working Group.
       </p>
     </section>
   </section>
-
+  
   <section>
-    <h2>Use Cases</h2>
-    <p>New cases to be aggregated here, from VC and DID Use Case documents.
-    </p>
+    <h2>User Roles</h2>
+      <p>
+        There are four roles supported by <a>Verifiable Credentials</a> as defined by the 
+        VCWG: <a>Issuer</a>,
+        <a>Verifier</a>, and <a>Holder</a>.
```

Comment:
Either the number changes as I have, or there's another role to name in the list...
```suggestion
        There are three roles supported by <a>Verifiable Credentials</a> as defined by the 
        VCWG: <a>Issuer</a>,
        <a>Verifier</a>, and <a>Holder</a>.
```